R/data.R

```diff
@@ -89,12 +89,16 @@ NULL
 #'   \item \code{sutter_habitat}: More details at \code{\link[DSMhabitat]{sutter_habitat}}
 #'   \item \code{yolo_habitat}: More details at \code{\link[DSMhabitat]{yolo_habitat}}
 #'   \item \code{delta_habitat}: More details at \code{\link[DSMhabitat]{delta_habitat}}
+#'   \item \code{spawn_decay_rate}: More details at \code{\link[DSMscenario]{spawn_decay_rate}}
```

Comment:
I linked these to `DSMscenario` but if we do not build a pkgdown site for `DSMscenario` they will not work.

Were we planning on doing that?

Reply:
sure, would you go ahead and do the yml for DSMscenario?

Reply:
will do